master/buildbot/buildslave/base.py

```diff
@@ -422,7 +422,7 @@ def disconnect(self):
 
         if self.conn is None:
             return defer.succeed(None)
-        log.msg("disconnecting old slave %s now" % self.slavename)
+        log.msg("disconnecting old slave %s now" % self.name)
```

Comment:
its part of IService:
https://github.com/twisted/twisted/blob/trunk/twisted/application/service.py#L86